src/main/java/xyz/duncanruns/julti/instance/MinecraftInstance.java

```diff
@@ -142,6 +150,13 @@ public void discoverInformation() {
         this.checkFabricMods();
 
         this.discoverName();
+
+        // check if fullscreen is true in standardoptions (bad)
+        if (GameOptionsUtil.tryGetStandardOption(this.getPath(), "fullscreen") == "true")
+        {
+            Julti.log(Level.WARN, this.getName() + " has fullscreen set to true in standardsettings!\r\n" +
```

Comment:
These instructions are gonna be inaccurate
Please specify opening standard manager, and also specify what the user is pressing yes to